account_entries_report_hooks/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/license-AGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl.html
+   :alt: License: AGPL-3
+
+===============================
+Account Entries Report Hooks
+===============================
+
+This module extends the functionality of account.entries.report module
+which intends to override it in order to support and allow you to extend
+it by other modules that want to add or change the standard behaviour.
+
```

Comment:
It is not a module, but a model. Please replace with "This module extends the functionality of the Account Entries Report" by overriding the standard query with a new one that will allow other modules to extend more flexibly.